gradio_app.py

```diff
@@ -101,6 +104,19 @@ def load_models(whisper_backend_name, whisper_model_name, alignment_model_name,
     if not os.path.exists(encodec_fn):
         os.system(f"wget https://huggingface.co/pyp1/VoiceCraft/resolve/main/encodec_4cb2048_giga.th")
 
+    # Define the source and destination paths
+    source_path = "/content/encodec_4cb2048_giga.th"
+    destination_path = "/content/VoiceCraft-gradio-colab/pretrained_models/encodec_4cb2048_giga.th"
```

Comment:
/content/ is only suitable colab, could you also make it compatible for if people run it locally?

Reply:
this issue is solved in https://github.com/jasonppy/VoiceCraft/pull/104